src/ranking/effectiveness/mark.ts

```diff
@@ -8,12 +8,12 @@ import {Schema} from '../../schema';
 
 import {FeatureScore} from '../ranking';
 import {getFeatureScore} from './effectiveness';
-import {BIN_Q, TIMEUNIT_T, TIMEUNIT_O, Q, N, O, T, NONE, ExtendedType, getExtendedType} from './type';
+import {BIN_Q, TIMEUNIT_T, TIMEUNIT_O, Q, N, O, T, NONE, getExtendedType} from './type';
```

Comment:
Looks like we have another set of incorrect merge conflicts in this file.
(In general, when you review your changes after merging conflict, if there are lines that you didn't add yourselves, that's sign of incorrect merge conflicts.)